library/bignum_core.c

```diff
@@ -33,11 +33,18 @@
 #include "bn_mul.h"
 #include "constant_time_internal.h"
 
+/**
```

Comment:
This is not a `static` function, so its documentation is in a header file (in this case `library/bignum_core.h` since it's not part of the public API). Please update the public documentation, and don't add a copy here (copies are bad because they get out of sync — this one is already out of sync!).